core/client/src/main/java/alluxio/client/file/options/SetAttributeOptions.java

```diff
@@ -146,11 +154,15 @@ public SetAttributeOptions setTtlAction(TtlAction ttlAction) {
   }
 
   /**
+   * @deprecated the persisted attribute is deprecated since version 1.4 and will be removed in 2.0
    * @param persisted the persisted flag value to use; it specifies whether the file has been
    *        persisted in the under file system or not.
    * @return the updated options object
    */
+  @Deprecated
   public SetAttributeOptions setPersisted(boolean persisted) {
+    LOG.warn("Setting the value of the persisted attribute is deprecated since version 1.4 and "
+        + "will be removed in 2.0.");
```

Comment:
Is this necessary since we already have the deprecated annotation? If you do not have this log warn, you can probably avoid the unit test pains